components/_global-navigation.scss

```diff
@@ -264,24 +277,65 @@ $notification-counter-size: 18px;
 		text-align: center;
 	}
 
-	@media #{$breakpoint-tablet-down} {
-		&__account-navigation-caption {
-			display: none;
+	&__start-a-wiki.wds-is-squished {
+		padding: 8px 10px;
+	}
+
+	&__start-a-wiki-caption {
+		display: none;
+	}
+
+	&__start-a-wiki-icon {
+		display: block;
+		height: 14px;
+		width: 14px;
+	}
+
+	@media #{$breakpoint-desktop-up} {
+		&__content-bar {
+			width: 1024px;
 		}
 
-		&__start-a-wiki::before {
-			content: '┼';
-			display: inline;
+		&__search {
+			flex: 2;
+
+			&-input-wrapper {
+				border-bottom: 1px solid $color-slate-gray;
+			}
+
+			&-input {
+				display: block;
+			}
```

Comment:
I see `-input-wrapper` and `-input` styles repeated in active state and on this breakpoint. What do you think about overriding this style only for mobile not active state?